docs/products/postgresql/concepts/high-availability.md

```diff
@@ -30,14 +30,18 @@ Aiven's Business and Premium plans offer
 
 ## Failure handling
 
-**Minor failures**, such as service process crashes or temporary loss of
+### Minor failures
+
+Minor failures, such as service process crashes or temporary loss of
 network access, are handled automatically by Aiven in all plans without
 any major changes to the service deployment. The service automatically
-restores normal operation once the crashed process is automatically
-restarted or when network access is restored.
+restores normal operation once the crashed process is restarted or when network access is
+restored.
+
+### Severe failures
 
-**Severe failures**, such as losing a node entirely in case of hardware
-or severe software problems, require drastic recovery measures. The
+Severe failures, such as losing a node entirely in case of hardware
+or severe software problems, require involved recovery measures. The
```

Comment:
The previous drastic word might sound a little bit "drastic" :) but I don't fully get the involved word in exchange. Both minor and severe failures are automatically handled by the system and they don't require any involvement from the user. The primary difference is during the severe failures, the system might take more serious measures to replace a node, on behalf of the user -- hence the "drastic" part.

Reply:
Right, no user involvment, but more time consuming as changing the DNS might take time (would be interesting to talk about this), and the presence of replication lag might also delay things and make things more complicated to recover.